dll/3rdparty/libpng/CMakeLists.txt

```
@@ -27,7 +27,7 @@ list(APPEND SOURCE
     pngwutil.c
     pngpriv.h)
 
-add_library(libpng MODULE ${SOURCE})
+add_library(libpng SHARED ${SOURCE})
```

```suggestion
spec2def(libpng.dll libpng.spec ADD_IMPORTLIB)

add_library(libpng MODULE ${SOURCE})
```

[libpng_spec.zip](https://github.com/user-attachments/files/18683126/libpng_spec.zip)